frontend/src/app/components/Leaderboards/LeaderboardsRow.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { withRouter } from 'react-router-dom'
+import {
+  TextSpan,
+  ListItem,
+  StyledLink,
+} from '../../../styles/blocks'
+import { createProfilePath } from '../../const/routes'
+
+const trophies = [
+  require('../../../media/trophy-1.svg'),
+  require('../../../media/trophy-2.svg'),
+  require('../../../media/trophy-3.svg'),
+]
+
+const LeaderboardsRowComponent = ({ user, position, points }) => (
+  <ListItem>
+    <TextSpan align="left">{
+      position > 3 ? (position + '.') : <img src={trophies[position - 1]} />
+    }</TextSpan>
+    <TextSpan textAlign="right">
+      <StyledLink to={createProfilePath(user.id)}>{user.name} ({points})</StyledLink>
+    </TextSpan>
+  </ListItem>
+)
+
+export const LeaderboardsRow = withRouter(LeaderboardsRowComponent)
```

Comment:
why is it wrapped in withRouter? I might be missig something

Reply:
I took this part from the `TopRatingRow` component where it is redundant as well. Will fix it.

Reply:
Fixed [here](https://github.com/salsita/foosball-rating/pull/92/commits/d7c17da31ec421acbb90f4a19ce22c34412cdf35#diff-ccbac6f012a8b829dcf10aa31411e483) and [here](https://github.com/salsita/foosball-rating/pull/92/commits/d7c17da31ec421acbb90f4a19ce22c34412cdf35#diff-32d0948dd609721b4bda2c94363a08eb)